### PR TITLE
Remove double scrollbars

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1577

### CODE REVIEW

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
This PR goes counter to what the goal was:
- reduce injected class names
- add few but well controlled props like variant to control appearance

Please read:
https://doesitsparkle.slack.com/archives/C02DPU9KF1A/p1638372815413200

File: src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx [hunk 1]
```diff
@@ -5,18 +5,12 @@ import "./AdminShowcase.scss";
 
 export interface AdminShowcaseProps {
   className?: string;
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx:7]:
```suggestion
```

File: src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx [hunk 2]
```diff
@@ -5,18 +5,12 @@ import "./AdminShowcase.scss";
...
   children,
 }) => {
-  const containerClasses = classNames(
-    "AdminShowcase",
-    className,
-    `AdminShowcase--${variant}`
-  );
+  const containerClasses = classNames("AdminShowcase", className);
   return <div className={containerClasses}>{children}</div>;
 };
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx:16]: This is all that is needed. With this (and removing the unused `className` from where it is used) I think this PR could be merged:

```tsx
export const AdminShowcase: React.FC = ({ children }) => (
  <div className="AdminShowcase">{children}</div>
);
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
After reviewing it again, and running it (without the `className`), only a small difference remains:
scrollbar in dashboard is constrained inside the components while the other pages have it on the whole document.

sparkle-scrolls-02
sparkle-scrolls-01

That difference is nice to have, not a showstopper, however, would approve after removing the `className` from `<AdminShowcase>`

File: src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx:16]: Also, a minor change would be to simply remove `&__main` in `RunTabView.scss` . Page works OK without it now.

File: src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx (hunk 1 above)
Comment (sunny-viktoryia) [reply src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx]: removed: https://github.com/sparkletown/sparkle/pull/2697/commits/78a039d05022790d3f2b8a12ef645a38ca956836

File: src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx (hunk 2 above)
Comment (sunny-viktoryia) [reply src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx]: removed and moved some styles in the `RunTabView.scss` to have the same view: https://github.com/sparkletown/sparkle/pull/2697/commits/78a039d05022790d3f2b8a12ef645a38ca956836

**Review by goran-peoski-work — APPROVED** [review]
LGTM

We can take care of unifying the styles in next round of IA adjustments

**Pull request merged**